docs/pywbemclicommands.rst

```diff
@@ -34,10 +34,10 @@ with mock files that are located in the pywbemtools ``tests/unit`` subdirectory.
 Class command group
 -------------------
 
-The **class** command group defines commands that act on CIM classes. see
-:ref:`pywbemcli class --help`. This group includes the following commands:
+The **class** command group defines commands that act on CIM classes. See
+:ref:`pywbemcli class --help`.
 
-This group consists of the following commands:
+This group includes the following commands:
```

Comment:
I picked consists of because it means that this is the complete list.  Includes implies that there may be others not listed.

Reply:
Agreed.

Reply:
DONE - Changed to "This group has the following commands:".